src/lib/utils/helpers.ts

```diff
@@ -94,9 +94,9 @@ export function humanizeUnixTimestamp(unixTimestamp: number): string {
   return new Date(unixTimestamp).toLocaleDateString('en-US');
 }
 
-export function humanizeDuration(duration: number) {
-  if (duration === 0) return '0 seconds';
-  return moment.duration(duration / 1000).humanize({ d: 7, w: 4 });
+export function humanizeDuration(durationMs: number) {
+  if (durationMs === 0) return '0 seconds';
+  return moment.duration(durationMs).humanize({ d: 7, w: 4 });
```

Comment:
@katien  --- Looks like this function is being used by an embalmer step that was removed from UI -- `SetDiggingFees` -- maybe both this function and that step code (which still remains) should be removed?

Reply:
Creating another task to remove dead code because we've got a significant amount in this repo